### PR TITLE
Add blog post about omicron lineage detection

### PR DESCRIPTION
**Pull request by wm75**

### CODE REVIEW

**Comment by wm75** [comment]
@bwlang presumably a very motivating example anaylsis to work through for the workshop

File: _posts/2021-11-29-omicron-and-galaxy.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+site: freiburg
+tags: [data, COVID-19]
+title: Omicron and SARS-CoV-2 genome surveillance
+supporters:
+- denbi
+- elixir
```
Comment (beatrizserrano) [inline _posts/2021-11-29-omicron-and-galaxy.md:7]: converge and by-covid?

File: _posts/2021-11-29-omicron-and-galaxy.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+[The full analysis history](https://usegalaxy.eu/u/sars-cov2-requests-bot/h/b11529-run234-midnight-primers) of the batch of 68 samples, most of which turn out to be of the Omicron lineage, enables everybody to verify claims made about the mutations found in the new VOC. No need to rely on hard to verify FASTA consensus sequences, nor on mainstream media - just check out the analysis and see the samples and the derived variant calls and lineage assignments right there.
+
+For the impatient, here's the summary plot of the variant spectrum of the batch (taken directly from the linked history above):
+
+![Omicron seen through Galaxy](/assets/media/2021-11-29-omicron-variant-plot.png)
+
+The reported, unprecedentedly high number of mutations in the spike protein (delineated by the narrow violet bar at the top of the plot) of the VOC can clearly be seen (and the full analysis confirms all key mutations that have been publically reported).
```
Comment (beatrizserrano) [inline _posts/2021-11-29-omicron-and-galaxy.md:23]: something sounds strange to me here, maybe the comma?

**Review by beatrizserrano — COMMENTED** [review]
nice one @wm75  :)

File: _posts/2021-11-29-omicron-and-galaxy.md (hunk 1 above)
Comment (bgruening) [reply _posts/2021-11-29-omicron-and-galaxy.md:7]: added

File: _posts/2021-11-29-omicron-and-galaxy.md (hunk 2 above)
Comment (bgruening) [reply _posts/2021-11-29-omicron-and-galaxy.md]: better?

**Comment by bgruening** [comment]
lets go with this, we can change as we go! Thanks a lot!

**Comment by wm75** [comment]
Too fast for me, but thanks!

**Pull request merged**